### ISSUE TITLE
v2.2.0 does not log errors to Rollbar

### ISSUE THREAD

**Issue by playerrtwo** [rollbar/rollbar-php-laravel#20]
laravel/framework 5.4.36
rollbar/rollbar-php-laravel 2.2.0

As soon as I get the update for 2.2.0 nothing gets logged to Rollbar. If I revert back to 2.1, everything works fine.

**Comment by drbyte** [comment]
1. Can you share your `composer.json` contents?
2. In your `.env` file, which ROLLBAR_XXXXXX keys are listed? (don't post their values, just the keys, like `ROLLBAR_ABC`, `ROLLBAR_FOO`, `ROLLBAR_BAZ`)
3. What operating system is your site running on? (this affects how the `get_env` stuff works)
4. Can you post the `rollbar` section of your `config/services.php`?

**Comment by playerrtwo** [comment]
`{
    "name": "laravel/laravel",
    "description": "Agent Legend",
    "keywords": ["framework", "laravel"],
    "license": "proprietary",
    "type": "project",
    "require": {
        "php": ">=5.5.9",
        "laravel/framework": "5.4.*",
        "phpmailer/phpmailer": "5.2.*",
        "twilio/sdk": "^4.5",
        "aws/aws-php-sns-message-validator": "^1.1",
        "guzzlehttp/guzzle": "~6.0",
        "dompdf/dompdf": "0.7.x@beta",
        "soundasleep/html2text": "~0.2",
        "recurly/recurly-client": "2.8.*",
        "doctrine/dbal": "^2.5",
        "pda/pheanstalk": "^3.1",
        "pusher/pusher-php-server": "^2.6",
        "laravel/passport": "^1.0",
        "phansys/getid3": "^1.9",
        "laravel/browser-kit-testing": "^1.0",
        "webpatser/laravel-uuid": "2.*",
        "intercom/intercom-php": "^3.1",
        "rollbar/rollbar-laravel": "^2.1",
        "fedeisas/laravel-mail-css-inliner": "^2.2"
    },
    "require-dev": {
        "fzaninotto/faker": "~1.4",
        "mockery/mockery": "0.9.*",
        "phpunit/phpunit": "~5.7",
        "symfony/css-selector": "2.8.*|3.0.*",
        "symfony/dom-crawler": "2.8.*|3.0.*"
    },
    "autoload": {
        "classmap": [
            "database"
        ],
        "psr-4": {
            "App\\": "app/",
            "ContactLegend\\": "app/ContactLegend"
        }
    },
    "autoload-dev": {
        "classmap": [
            "tests/TestCaseBase.php"
        ],
        "psr-4": {
            "App\\Tests\\": "tests/"
        }
    },
    "scripts": {
        "post-root-package-install": [
            "php -r \"copy('.env.example', '.env');\""
        ],
        "post-create-project-cmd": [
            "php artisan key:generate"
        ],
        "post-install-cmd": [
            "php artisan clear-compiled",
            "php artisan optimize",
            "bower install",
            "php artisan migrate --force"
        ],
        "post-update-cmd": [
            "php artisan optimize",
            "bower install",
            "php artisan migrate --force",
            "php artisan queue:restart"
        ]
    },
    "config": {
        "preferred-install": "dist"
    }
}
`

**Comment by playerrtwo** [comment]
2. No ROLLBAR_XXXXX keys in my .env file

3. My local development environment is Homestead (Ubuntu 16.04) and my production environment is CentOS 6.9. Both environments experience the error.

4. `'rollbar' => [
        'access_token' => '00b6cad33afb4793b5862274ff6465f5',
        'environment' => env('APP_ENV'),
        'level' => env('ROLLBAR_LEVEL', 'info'),
        'root' => base_path(),
        'person_fn' => function() {
            return App\Utility\RollbarUtility::personFn();
        }
    ]`

**Comment by drbyte** [comment]
> No ROLLBAR_XXXXX keys in my .env file

Okay. There's been a small bugfix since 2.2.0 that hasn't been tagged yet.

If you add your ROLLBAR_TOKEN to `.env` it'll probably fix it:
`ROLLBAR_TOKEN={your token here}`

Alternatively, you could edit your `config/services.php` and duplicate the `access_token` line to show `token` too, like this:

```diff
'rollbar' => [
        'access_token' => '{my access token}',
+// temporary bugfix:
+       'token' => '{my access token}',
        'environment' => env('APP_ENV'),
```

**Comment by ArturMoczulski** [comment]
This has been included in tag v2.2.1

Let us know if you still have problems.

@drbyte Thank you for help with this
